source/Tfe/DNS.cpp

```diff
@@ -25,6 +25,9 @@ Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA  02111-1307  USA
 #ifndef _MSC_VER
 #include <arpa/inet.h>
 #include <netdb.h>
+#ifdef __FreeBSD__
+#include <sys/socket.h> // AF_INET.
```

Comment:
no need to add `ifdef`, I will commit this.

Reply:
Agree with that. This is just to explain the PR.
Should I update my PR? Or would you just fix your code in your way and drop the PR?
That's totally fine.